src/person/controllers/mass_message_controller.py

```diff
@@ -58,7 +58,7 @@ def process(self):
         if sandbox:
             # no messages were sent
             return success_response(
-                data=f"{notifs_enabled_count} users have push notifications enabled",
+                data=f"{notifs_enabled_count} users have push notifications enabled, no notfications sent (sandbox mode)",
```

Comment:
this doesn't look related to this PR, what is 'sandbox mode'?

Reply:
ah this is a smol change, sandbox mode allows us to see how many people have notifs enabled without actually sending a notif to everyone, I recently added it to our API spec